olp-cpp-sdk-authentication/include/olp/authentication/CredentialsHelper.h

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Done.
Changed return type to `boost::optional<AuthenticationCredentials>` because `client::ApiResponse` doesn't accept types that has no default constructor.